company/about-mattermost/list-of-terms.md

```diff
@@ -309,3 +309,27 @@ Previously hyphenated, now not hyphenated, see [Tombstoned](list-of-terms.md#tom
 
 A team and process that is ready for a key member to go on vacation, or spend time away from work.
 
+#### Mattermost Cloud
+
+Describes cloud infrastructure within AWS, where the Cloud Provisioning server, Mattermost Operator, databases, file storage, etc exist. 
+
+#### Installation
+
+An installation is an instance of Mattermost, otherwise known as a Mattermost [Workspace](#Workspace). Internally, the term Installation generally implies it is hosted inside Mattermost Cloud
```

Comment:
```suggestion
An installation is an instance of Mattermost, otherwise known as a Mattermost [Workspace](#Workspace). Internally, the term Installation generally implies it is hosted inside Mattermost Cloud.
```